docs/gallery-submission.md

```diff
@@ -17,116 +15,110 @@ title: Connector Tableau Exchange Submission Guide
+1. Visit the [Tableau Exchange](https://exchange.tableau.com/) website.
+2. Click **Sign In** in the upper-right corner.
+3. If you already have a Tableau.com account, sign in with that account. At this time, signing in with Salesforce is not supported. If you don’t have a Tableau.com account, click **Create an Account**. To make it easier to verify your account, enter the same email address when signing up for a Tableau.com account, the Tableau Partner Network, and, if you have one, a Salesforce account.
 
-### Step 2: Build your integration
+### Step 2: Build and test your connector
 
-After you've signed up for our developer program, use the information in this Connector SDK documentation to get started building a connector. If you have issues with the SDK or need help, submit an issue through our [GitHub issues page](https://github.com/tableau/connector-plugin-sdk/issues) and include the [required artifacts](https://tableau.github.io/connector-plugin-sdk/docs/bug-artifacts) with the issue.
+Use the information in the [Connector SDK documentation]({{ site.baseurl }}/docs/) to build your connector. If you have issues with the SDK or need help, submit an issue through our [GitHub issues page]({{ site.baseurl }}/issues) and include the [required artifacts]({{ site.baseurl }}/docs/bug-artifacts) with the issue.
 
-Be sure to test your connector with the [TDVT suite](https://tableau.github.io/connector-plugin-sdk/docs/tdvt) to ensure that your connector meets our guidelines for connector functionality.
+Be sure to test your connector with [the TDVT suite]({{ site.baseurl }}/docs/tdvt) to ensure that your connector meets our guidelines for connector functionality.
 
 **Note:** The Tableau Exchange no longer accepts submissions of ODBC connectors. Only JDBC connectors are accepted.
 
-## Submitting Your Connector to Tableau Exchange
-
-### Step 3: Sign up for Technology Partner Program
-
-After you've built your integration and are passing 90% of TDVT results, sign up for our [Technology Partner Program](https://www.tableau.com/partners/resources/technology/partner-agreement). By signing up for the Technology Partner Program you allow us to discuss confidential information with you and enables us to promote your products through our marketing channels, like Tableau Exchange.
-
-### Step 4: Submit a Request to Publish
-
-After you've built and tested your integration, [submit a request](https://forms.gle/2FisXfmwwuQEaXKG9) to publish your connector in Tableau Exchange.
+### Step 3: Join the Tableau Partner Network
 
-### Step 5: Sign Tableau Exchange Agreement
+After you’ve built your integration and are passing 90% of TDVT results, join the Technology Track on our [Tableau Partner Network (TPN)](https://www.tableau.com/partners/become). By joining the TPN, you allow us to discuss confidential information with you and enable us to promote your products through our marketing channels, like the Tableau Exchange.
```

Comment:
Can you also add this beside 90% of TDVT results- (all required tests should pass or acknowledged as not supported or known bug for the underlying data source).